src/main/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestClientInternal.java

```diff
@@ -493,7 +493,6 @@ ChannelsStatusResponse getChannelsStatus(
               .collect(Collectors.toList());
       request.setChannels(requestDTOs);
       request.setRole(this.role);
-      request.setRequestId(this.flushService.getClientPrefix() + "_" + counter.getAndIncrement());
```

Comment:
We should remove request_id in the request as well

Reply:
you mean in the POJO?

Reply:
Yes